manuscript/Hacking.md

```diff
@@ -45,7 +47,12 @@ This generation of hackers will create technology to sell PT products around the
  
 The generation that was hacking Linux boxes while we were in university is the same generation that is solving fundamental problems today. 
 
-We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one. There is a factory in Portugal where this can be done.
+We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one (see https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit/). There should be a factory in Portugal that can do this.  
```

Comment:
Links don't print out well. 
Replace with a comment about how it has been a successful commercial success with  [Ten Millionth Raspberry Pi](https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit)  sold, which ironically is the population of Portugal. The Raspberry Pi team has made an effort to build them in UK (which is where the reference to a Factory in PT make sense)